src/main/java/org/example/Functions.java

```diff
@@ -2,5 +2,49 @@
 
 public class Functions {
 
+    public static void bubbleSort(int[] mas){
+        int n = mas.length;
 
+        for (int i = 0; i < n - 1; i++){
+            for (int j = 0; j < n - i - 1; j++){
+                if(mas[j] > mas[j+1]){
+                    int vr = mas[j];
+                    mas[j] = mas[j+1];
+                    mas[j+1] = vr;
+                }
+            }
+        }
+    }
+
+
+    public static int binarySearch(int[] mas, int el){
```

Comment:
consider using more meaningful variables naming. array or arr instead of mas. element instead of el

Reply:
ok